qcodes/instrument_drivers/Keithley/_Keithley_2600.py

```diff
@@ -626,6 +630,13 @@ def doFastSweep(self, start: float, stop: float, steps: int, mode: str) -> DataS
                 'VI' (current sweep two probe setup) or
                 'VIfourprobe' (current sweep four probe setup)
         """
```

Comment:
so decided not to deprecate doFastSweep method?

Reply:
It does not have a replacement yet https://github.com/QCoDeS/Qcodes/issues/4976